site/docs/build-ref.html

```diff
@@ -298,6 +298,18 @@ <h3 id="labels">Labels</h3>
   <code>//my/app/testdata:testdepot.zip</code>.
 </p>
 
+<p>
+  Labels starting <code>@//</code> are references to the main
+  repository, which will still work even from external repositories.
+  Therefore <code>@//a/b/c</code> is different from
+  <code>//a/b/c</code> when referenced from an external repository;
```

Comment:
Nit: I'd  just make this a clean sentence break to keep each sentence easily digestable..